### PR TITLE
Fix error check of a routine execution

### PR DESCRIPTION
**Pull request by tfaller**
Because a routine execution is done by a multi query, all results need to be checked for errors.

Signed-off-by: Thomas Faller tfaller1@gmx.de

### CODE REVIEW

File: libraries/rte/rte_routines.lib.php [hunk 1]
```diff
@@ -1416,7 +1416,10 @@ function PMA_RTN_handleExecute()
 
                 $GLOBALS['dbi']->freeResult($result);
 
-            } while ($GLOBALS['dbi']->nextResult());
+            } while ($outcome = $GLOBALS['dbi']->nextResult());
```
Comment (nijel) [inline libraries/rte/rte_routines.lib.php]: $outcome will be always false at the end of this loop, is this really what you wanted to achieve?

Comment (tfaller) [inline libraries/rte/rte_routines.lib.php]: That is not the case.

https://github.com/tfaller/phpmyadmin/blob/037bddd7c3560c1f0e81730d815302f67f98d5bc/libraries/rte/rte_routines.lib.php#L1411

The loop will be left by a `break` if no more results are available.
$outcome will be only false if `$GLOBALS['dbi']->moreResults()` was true and an error occured.

Comment (nijel) [inline libraries/rte/rte_routines.lib.php]: Ah okay, sorry for confusion.

**Comment by nijel** [comment]
Merged, thanks!

**Pull request merged**